airflow/models/trigger.py

```diff
@@ -175,7 +175,7 @@ def assign_unassigned(cls, triggerer_id, capacity, session=None):
         alive_triggerer_ids = [
             row[0]
             for row in session.query(BaseJob.id).filter(
-                BaseJob.end_date is None,
+                BaseJob.end_date == None,  # noqa: E711
```

Comment:
```suggestion
                BaseJob.end_date.is_(None),
```
this looks a little better to me... and i _think_ we can do, no?

Reply:
thanks, agreed